internal/cmdrender/cmdrender.go

```diff
@@ -52,6 +52,8 @@ func NewRunner(ctx context.Context, parent string) *Runner {
 		fmt.Sprintf("pull image before running the container. It must be one of %s, %s and %s.", fnruntime.AlwaysPull, fnruntime.IfNotPresentPull, fnruntime.NeverPull))
 	c.Flags().BoolVar(&r.allowExec, "allow-exec", false,
 		"allow binary executable to be run during pipeline execution.")
+	c.Flags().BoolVar(&r.allowWasm, "allow-wasm", false,
+		"allow wasm to be used during pipeline execution.")
```

Comment:
The help message should probably specify that this feature is alpha

Reply:
Updated it to `allow-alpha-wasm`